api/API_VERSIONING.md

```diff
@@ -28,9 +29,24 @@ configuration resources (e.g. bootstrap, xDS resources such as `Cluster`). The
 v3 API bootstrap configuration is `envoy.config.bootstrap.v3.Boostrap`, even
 though it might might transitively reference `envoy.service.trace.v2`.
 
+The minor version enables API stability and support, and to safely deprecate old
+features and reduce the tech debt. A client can support a range of minor API
+versions, and can request xDS resources that match this range, using the resource
+[context parameters](https://github.com/cncf/udpa/blob/master/xds/core/v3/context_params.proto).
```

Comment:
Also `Node` will indicate for simple string names. I'd generally lead in with the old way of doing it and then point out the new way.

Reply:
Added `user_agent_latest_api_version` and `user_agent_oldest_api_version` to `Node`.